extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -216,6 +218,27 @@ export class ProjectsController {
 		}
 	}
 
+	public async readPublishProfile(profileUri: Uri): Promise<PublishSettings> {
+		const profileText = await fs.readFile(profileUri.fsPath);
+		const profileXmlDoc = new xmldom.DOMParser().parseFromString(profileText.toString());
+
+		// read target database name
+		let targetDbName: string = '';
+		let targetDatabaseNameCount = profileXmlDoc.documentElement.getElementsByTagName(constants.targetDatabaseName).length;
+		if (targetDatabaseNameCount > 0) {
+			// if there is more than one TargetDatabaseName nodes, SSDT uses the name in the last one so we'll do the same here
```

Comment:
hmm.. good catch. Why are more than one db names allowed in SSDT?

Reply:
Not sure. It's pretty strange :/